copyIOSLibs.sh

```diff
@@ -1,5 +1,4 @@
 #!/bin/bash
-cp -f ./xamarin/FHXamarinIOSSDK/bin/Release/FHSDK.dll ./Dist/lib/xamarinios10/
-#cp -f ./xamarin/FHXamarinIOSSDK/bin/Release/FHSDK.xml ./Dist/lib/monotouch/
-cp -f ./xamarin/FHXamarinIOSSDK/bin/Release/FHXamarinIOSSDK.dll ./Dist/lib/xamarinios10/
-cp -f ./xamarin/FHXamarinIOSSDK/bin/Release/FHXamarinIOSSDK.XML ./Dist/lib/xamarinios10/
\ No newline at end of file
+cp -f ./xamarin/FHXamarinIOSSDK/bin/Debug/FHSDK.dll ./Dist/lib/xamarinios10/
```

Comment:
When running copyIOSLibs.sh => ERROR
./xamarin/FHXamaringIOSSDK/bin/debug/FHSDK.dll no such file or directory
./xamarin/FHXamaringIOSSDK/bin/debug/FHXamarinIOSSDK.dll no such file or directory

When using Visual Studio, I noticed binaries get into
FHXamaringIOSSDK/bin/debug folder

@edewit Is it different with Xamarin Studio on iOS (to be checked) so it could explained the different path??